lighthouse-plugin-publisher-ads/utils/resource-classification.js

```diff
@@ -132,6 +132,29 @@ function isGptTag(url) {
     'securepubads.g.doubleclick.net'].includes(host);
   const matchesPath =
     ['/tag/js/gpt.js', '/tag/js/gpt_mobile.js'].includes(pathname);
+  return ( matchesHost && matchesPath );
+}
+
+/**
+ * Checks if the url is loading a gpt.js, or an amp-ad-{version}.js script.
+ * @param {URL|string} url
+ * @return {boolean}
+ */
+function isGptOrAmpTag(url) {
```

Comment:
Nit: Consistent capitalization of Amp (line 143) and AMP (line 144). (Doesn't matter to me which one we choose, since we're already all-caps-ing URL but not GPT :/  )

Reply:
I'll go with All Caps AMP that is what the project seems to use.